cocos/platform/CCFileUtils.h

```diff
@@ -99,6 +160,87 @@ class CC_DLL FileUtils
      */
     virtual Data getDataFromFile(const std::string& filename);
 
+
+    enum class Error
```

Comment:
I think it is better to rename Result or something similar because the return type may be ok.